### ISSUE TITLE
Support compiling to wasm [blocked]

### ISSUE THREAD

**Issue by kevmoo** [gskinnerTeam/flutter-wonderous-app#135]
This is a tracking issue that is mostly blocked on other packages, but it's nice to have a roll-up.

- [x] file_selector_web
- [ ] google_maps_flutter_web
- [x] package_info_plus
- [x] shared_preferences_web
- [x] url_launcher_web
- [ ] youtube_player_iframe_web

Tracking details are at https://docs.google.com/spreadsheets/d/1ToXdnF2_Yucn6IpeTXldoUE6-8Su3FXRu-31GwSE_Fs/edit?resourcekey=0-DdqKaEwQdN9NZVllOs6ZgA#gid=0 – google-only at the moment.

@kevmoo is the main coordinator here!
